source/index.md

```diff
@@ -763,10 +931,13 @@ Declined   |  40000 |  General input error
 
 ### Status message
 
-A status message is included in every response when you make a new transaction.
+A status message may be included in the response when you create a new
+transaction.
 The status message can be used for debugging and may include a more specific
 error message.
 
+Status messages should be shown to the merchant.
```

Comment:
I think this highly depends on the message. I can imagine that most merchants don't want to announce e.g. they have hit a sales limit.

Reply:
The text says "should be shown to the merchant", not "should be shown to the cardholder".

Reply:
✅